client/README.md

```diff
@@ -0,0 +1,44 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Remove default react README and write a brief intro about the project and steps to prepare, test, etc.